cluster/src/main/java/io/scalecube/cluster/Cluster.java

```diff
@@ -178,7 +175,7 @@ static Cluster joinAwait(ClusterConfig config) {
    *
    * @return Listenable future which is completed once graceful shutdown is finished.
    */
-  CompletableFuture<Void> shutdown();
+  Mono<Void> shutdown();

```

Comment:
Plz dont substitute `futures` to reactor `publishers`. Let's do this in separate PR.